src/main/java/org/sagebionetworks/bridge/sdk/AdminClient.java

```diff
@@ -18,37 +18,44 @@
      *            Credentials used to sign up the user into Bridge.
      * @return true if success, false if failure.
      */
-    public boolean createUser(SignUpByAdmin signUp);
+    boolean createUser(SignUpByAdmin signUp);
```

Comment:
What does the javadoc look like if you reduce the visibility of this interface and this interface is all that is in the javadoc? Is it still there? Will it look different than the other two parallel interfaces? (I asked on Slack but you don't seem to see messages).